corehq/apps/app_manager/tests/data/suite/sync_request.xml

```diff
@@ -0,0 +1,19 @@
+<partial>
+  <sync-request>
+    <post url="https://www.example.com/a/test_domain/phone/claim-case/">
+      <data key="case_id" ref="session/datum/case[@id]"/>
+      <data key="case_type" ref="session/datum/case/type"/>
+    </post>
+    <session>
+      <query url="https://www.example.com/a/test_domain/phone/search/">
+        <prompt key="name">
+          <display>Name</display>
```

Comment:
the somethings are things that app builder makes up and reference values that get put into the app strings files. See `app_strings.py`